test_utils/utils.go

```diff
@@ -203,14 +209,21 @@ func DeleteContainer(netconf, netnspath, name string) (session *gexec.Session, e
 	subProcess := exec.Command("bash", "-c", fmt.Sprintf("%s dist/%s", cni_env, os.Getenv("PLUGIN")), netconf)
 	stdin, err := subProcess.StdinPipe()
 	if err != nil {
-		panic("some error found")
+		return
 	}
-
 	io.WriteString(stdin, netconf)
 	io.WriteString(stdin, "\n")
 	stdin.Close()
 
-	session, err = gexec.Start(subProcess, ginkgo.GinkgoWriter, ginkgo.GinkgoWriter)
+	session, err := gexec.Start(subProcess, ginkgo.GinkgoWriter, ginkgo.GinkgoWriter)
+	if err != nil {
+		return
+	}
+
+	// Wait should supposedly induce a test failure if it times out?
```

Comment:
Yea that was a tad lazy of me. I added a 6 second timeout and can confirm the test prints a failure in an obvious manner. Will update the comment.